src/pages/Discussions/DiscussionListPage.tsx

```diff
@@ -27,10 +23,23 @@ export const DiscussionListPage: FC<DiscussionsPageProps> = ({ paths }) => {
   const { communityName } = useCommunityContext();
   const { discussionList, loading, permissions } = useDiscussionsContext();
 
-  const [category, setCategory] = useState<string>();
+  const showAllTitle = t('common.show-all');
+
+  const categoryConfig = useMemo<CategoryConfig[]>(
+    () => [
+      { title: showAllTitle, icon: AllInclusive },
+      { title: t('common.enums.discussion-category.GENERAL'), icon: QuestionAnswerOutlined },
+      { title: t('common.enums.discussion-category.IDEAS'), icon: LightbulbOutlined },
+      { title: t('common.enums.discussion-category.QUESTIONS'), icon: HelpOutlined },
+      { title: t('common.enums.discussion-category.SHARING'), icon: ShareOutlined },
+    ],
+    [showAllTitle, t]
+  );
+
+  const [category, setCategory] = useState<string>(categoryConfig[0].title);
 
   const filtered = useMemo(
-    () => discussionList?.filter(d => category === CATEGORY_ALL || d.category === category),
+    () => discussionList?.filter(d => !category || category === showAllTitle || d.category === category),
```

Comment:
Moving `category === showAllTitle` out of `filter()` makes it slightly more effective and slightly less heavy in terms of readability (conditional expression is split into 2).
Sorry for not pointing that out earlier.